src/components/accessories/admin/vaccines/vaccinesTable/VaccinesTable.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from "react";
+import Table from "../../../table/Table";
+import { useTranslation } from "react-i18next";
+import InfoBox from "../../../infoBox/InfoBox";
+import { CircularProgress } from "@material-ui/core";
+import { useDispatch, useSelector } from "react-redux";
+import { IState } from "../../../../../types";
+import { VaccineDTO } from "../../../../../generated";
+import { IApiResponse } from "../../../../../state/types";
+import classes from "./VaccinesTable.module.scss";
+import { getVaccines } from "../../../../../state/vaccines/actions";
+import { TFilterField } from "../../../table/filter/types";
+
+export const VaccinesTable = () => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    dispatch(getVaccines());
+  }, [dispatch]);
+
+  const header = ["code", "type", "description"];
+
+  const label = {
+    code: t("vaccine.code"),
+    type: t("vaccine.vaccinetype"),
+    description: t("vaccine.description"),
+  };
+
+  const filters: TFilterField[] = [
+    { key: "type", label: t("vaccine.vaccinetype"), type: "text" },
```

Comment:
I would be better to handle "type" filter as a select, you could provide the vacines types as options.  You could also refer to existing filters.